app/partials/app.cjsx

```diff
@@ -24,9 +24,9 @@ PanoptesApp = React.createClass
     initialLoadComplete: false
     user: null
 
-  updateUser: (user) ->
-    @setState user: user
-
```

Comment:
`updateUser` is referenced on lines 14 and 20, too, mind removing those? Then we can close #2740 with this.